docker/containerbuild.sh

```diff
@@ -32,7 +32,17 @@ if [ -d "$BUILDROOT" ]; then
 fi
 
 # Fetch the boot.iso for the build.
-curl http://mirror.centos.org/centos/"${KSNAME##*-}"/os/x86_64/images/boot.iso -o /tmp/boot-"${KSNAME##*-}".iso
+if [[ ${CONT_ARCH} -eq "ppc64le" ]]; then
+    VERSION=`expr "${KSNAME}" : '.*\-\([0-9\.]*\).*'`
+    if [[ "${VERSION}" = "7" ]]; then
+        curl http://mirror.centos.org/altarch/"${VERSION}"/isos/ppc64le/CentOS-7-AltArch-ppc64le-NetInstall-1511.iso -o /tmp/boot-"${KSNAME##*-}".iso
```

Comment:
Its bit difficult to make it more generic as we don't have any common pattern in the iso urls. I'm thinking of introducing one more commandline which accepts the boot iso link. Let me know your opinion?